rllib/agents/dqn/dqn.py

```diff
@@ -153,33 +156,73 @@ def check_config_and_setup_param_noise(config):
                               config.get("n_step", 1))
     config["sample_batch_size"] = adjusted_batch_size
 
+    # Backward compatibility of epsilon-exploration config AND beta-annealing
+    # fraction settings (both based on schedule_max_timesteps, which is
+    # deprecated).
+    schedule_max_timesteps = None
+    if "schedule_max_timesteps" in config and \
+            config["schedule_max_timesteps"] > 0:
+        deprecation_warning(
+            "schedule_max_timesteps", "exploration.epsilon_timesteps AND "
+            "prioritized_replay_beta_annealing_timesteps")
+        schedule_max_timesteps = config["schedule_max_timesteps"]
+    if "exploration_final_eps" in config and \
+            config["exploration_final_eps"] > 0:
+        deprecation_warning("exploration_final_eps",
+                            "exploration.final_epsilon")
+        config["exploration"] = config.get("exploration",
+                                           {"type": EpsilonGreedy})
+        if isinstance(config["exploration"], dict):
+            config["exploration"]["final_epsilon"] = \
+                config.pop("exploration_final_eps")
+    if "exploration_fraction" in config and config["exploration_fraction"] > 0:
+        assert schedule_max_timesteps is not None
+        deprecation_warning("exploration_fraction",
+                            "exploration.epsilon_timesteps")
+        config["exploration"] = config.get("exploration",
+                                           {"type": EpsilonGreedy})
+        if isinstance(config["exploration"], dict):
+            config["exploration"]["epsilon_timesteps"] = config.pop(
+                "exploration_fraction") * schedule_max_timesteps
+    if "beta_annealing_fraction" in config and \
+            config["beta_annealing_fraction"] > 0:
+        assert schedule_max_timesteps is not None
+        deprecation_warning(
+            "beta_annealing_fraction (decimal)",
+            "prioritized_replay_beta_annealing_timesteps (int)")
+        config["prioritized_replay_beta_annealing_timesteps"] = config.pop(
+            "beta_annealing_fraction") * schedule_max_timesteps
+    if "per_worker_exploration" in config and \
+            config["per_worker_exploration"] != -1:
+        deprecation_warning(
+            "per_worker_exploration",
+            "exploration.type=PerWorkerEpsilonGreedy")
+        if isinstance(config["exploration"], dict):
+            config["exploration"]["type"] = PerWorkerEpsilonGreedy
```

Comment:
Can you make a PR to update the YAMLs in https://github.com/ray-project/rl-experiments to use the new configs instead of the deprecated ones?

(also, anything in the tuned examples or jenkins tests)

Reply:
Sure, I wanted to run one clean test first to make sure everything is backward compatible.

Reply:
tuned_examples, Jenkins tests: done